scripts/downlevel-dts/index.js

```diff
@@ -0,0 +1,91 @@
+// @ts-check
+const yargs = require("yargs");
+
+const { existsSync, readdirSync, readFileSync, statSync, writeFileSync } = require("fs");
+const { join } = require("path");
+const { execSync } = require("child_process");
+const stripComments = require("strip-comments");
+
+// ToDo: Write downlevel-dts as a yargs command, and import yargs in scripts instead.
+yargs
+  .usage(
+    "Runs downlevel-dts npm script (if present) in each workspace of monorepo," +
+      " and strips comments from *.d.ts files.\n\n" +
```

Comment:
Nit: add mentioning please make sure the packages are already compiled to .js files?